app/controllers/hyrax/admin/workflow_roles_controller.rb

```diff
@@ -0,0 +1,41 @@
+module Hyrax
+  module Admin
+    class WorkflowRolesController < ApplicationController
+      before_action :require_permissions
+      layout 'admin'
+
+      def index
+        # TODO: Find a better way to make sure workflows are loaded
```

Comment:
Maybe, but this PR is about enabling workflows in our current tenants. I have not written the words that will magically close the issue so we should revisit in the new year with a better solution.